husky_bringup/launch/bumblebee/bumblebee.launch

```diff
@@ -0,0 +1,38 @@
+<launch>
```

Comment:
When including it in this location and running husky_bringup install when the env var is set, this will spawn on boot.